COLLABORATOR_GUIDE.md

```diff
@@ -643,35 +643,19 @@ versions. You can find more information
 a branch enters LTS, the release plan limits the types of changes permitted in
 the branch.
 
-#### Landing semver-minor commits in LTS
-
-The default policy is to not land semver-minor or higher commits in any LTS
-branch. However, the LTS WG or TSC can evaluate any individual semver-minor
-commit and decide whether a special exception ought to be made. It is
-expected that such exceptions would be evaluated, in part, on the scope
-and impact of the changes on the code, the risk to ecosystem stability
-incurred by accepting the change, and the expected benefit that landing the
-commit will have for the ecosystem.
-
-Any Collaborator who feels a semver-minor commit should be landed in an LTS
-branch should attach the `lts-agenda` label to the pull request. The LTS WG
-will discuss the issue and, if necessary, will escalate the issue up to the
-TSC for further discussion.
```

Comment:
This paragraph (about labelling pull requests) isn't currently covered in https://github.com/nodejs/Release#release-plan. I'm not sure whether this should remain here for visibility or be added to the Release repository. 

The label is now [`Release-agenda`](https://github.com/nodejs/node/labels/Release-agenda).

Reply:
Yeah, at a minimum, the label would have to be updated. That said, I removed it because:

* I'm not sure adding a PR to the Release agenda is actually the preferred way anymore TBH.
* PRs in these cases will usually be closed (because they've landed in master). Adding the label won't have it show up on the Release WG agenda assuming they're using the usual tools that other Node.js WGs use because the tool ignores closed issues! So the advice is probably bad advice anyway. (Probably the best thing to do is to @-mention the release wg or open an issue in the release WG repo.)
* Whatever the correct info is, a good case can be made that it be added in the Release Plan rather than here? Like, stuff about how to interact with the Release WG probably belongs in the Release WG repo?